src/main/java/com/google/cloud/spanner/pgadapter/statements/SessionStatementParser.java

```diff
@@ -361,6 +386,21 @@ static ShowStatement parseShowStatement(SimpleParser parser) {
       }
       return ShowStatement.createShowAll();
     }
+
+    if (parser.eatKeyword("time", "zone")) {
```

Comment:
Same here: I think it is good that we detect the special `show time zone` like this, but we can probably just add an `else` to read the variable name + check for `all` for all other statements, and after that keep the two branches together.